src/test/kotlin/no/skatteetaten/aurora/gobo/integration/DefaultHeadersTest.kt

```diff
@@ -4,30 +4,30 @@ import assertk.assertThat
 import assertk.assertions.isEqualTo
 import no.skatteetaten.aurora.gobo.ApplicationConfig
 import no.skatteetaten.aurora.gobo.HEADER_KLIENTID
-import no.skatteetaten.aurora.gobo.ObjectMapperConfig
+import no.skatteetaten.aurora.gobo.TestConfig
 import no.skatteetaten.aurora.mockmvc.extensions.mockwebserver.execute
+import no.skatteetaten.aurora.mockmvc.extensions.mockwebserver.url
 import okhttp3.mockwebserver.MockResponse
 import okhttp3.mockwebserver.MockWebServer
 import org.junit.jupiter.api.Test
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.bodyToMono
 
-@SpringBootTest(classes = [ApplicationConfig::class, ObjectMapperConfig::class])
+@SpringBootTest(classes = [ApplicationConfig::class, TestConfig::class])
 class DefaultHeadersTest {
 
     private val server = MockWebServer()
     private val url = server.url("/")
 
     @Autowired
-    private lateinit var applicationConfig: ApplicationConfig
+    private lateinit var webClient: WebClient
 
     @Test
     fun `Verify KlientID header`() {
-        val webClient = applicationConfig.webClientBuilder(false).baseUrl(url.toString()).build()
-
         val request = server.execute(MockResponse()) {
-            webClient.get().retrieve().bodyToMono<Unit>().block()
+            webClient.get().uri(server.url).retrieve().bodyToMono<Unit>().block()
         }
 
         val headers = request.first()?.headers
```

Comment:
burde vi testet for korrelasjonsid i tillegg?

Reply:
Ja, helt enig